api/model_template_summary_variable_args_render.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+func (args *TemplateSummaryVariableArgs) String() string {
+	if args == nil {
+		return "<nil>"
+	}
+	switch args.Type {
+	case "map":
+		b, err := json.Marshal(args.Values)
+		if err != nil {
+			return "{}"
+		}
+		return string(b)
+	case "constant":
+		values, ok := args.Values.([]interface{})
+		if !ok {
+			return "[]"
```

Comment:
Hmm feels like you might want to log something for these to indicate that the type casting didn't work? Unless it's expected to be `!ok` normally? (That would be kind of strange..?)